README.Rmd

```diff
@@ -13,7 +13,7 @@ knitr::opts_chunk$set(
 )
 ```
 
-# epiparameter
+# epiparameter <img src="man/figures/epiparameter_logo.png" align="right" height="200" />
 
 <!-- badges: start -->
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
I know updated vignettes etc. is next on list, but would be really useful to have a quick start in README with the updated syntax for pulling a parameter (e.g. pmf) out of library and defining as useable object.

Reply:
Resolved in 608ffecf13523ae0e18cc333e98745e1597404f0